packages/chrome/src/views/header/Header.example.md

```diff
@@ -1,9 +1,9 @@
 ```jsx
 const { Toggle, Label } = require('@zendeskgarden/react-toggles/src');
 const SupportIcon = require('svg-react-loader?name=Settings!@zendeskgarden/svg-icons/src/26/relationshape-support.svg');
-const HelpIcon = require('svg-react-loader?name=Settings!@zendeskgarden/svg-icons/src/14/support.svg');
-const MenuTrayIcon = require('svg-react-loader?name=Settings!@zendeskgarden/svg-icons/src/14/menu-tray.svg');
-const PersonIcon = require('svg-react-loader?name=Settings!@zendeskgarden/svg-icons/src/14/person.svg');
+const HelpIcon = require('svg-react-loader?name=Help!@zendeskgarden/svg-icons/src/16/lifesaver-stroke.svg');
```

Comment:
A number of these modifications are premature as the component CSS styling hasn't yet been updated to properly accommodate 16px icons. As a result, the icons appear under-sized/weighted. We should probably schedule this PR to happen after the CSS updates.